javascript/test/GherkinInMarkdownTokenMatcherTest.ts

```diff
@@ -186,4 +209,21 @@ describe('GherkinInMarkdownTokenMatcher', function () {
     ]
     assert.deepStrictEqual(t.matchedItems, expectedItems)
   })
+
+  it('matches arbitrary text as Empty after the FeatureLine has already been matched', () => {
```

Comment:
This is testing implementation detail, but I find it useful documentation for the token matcher - there is inherent state management here that I found confusing to begin with (resulting in a bug in my Python translation)

This behaviour is captured under the acceptance tests, but was useful for me for understanding where differences between implementations were.

Thoughts - is this useful to keep, or preferable to remove?